api/src/test/java/uk/gov/dhsc/htbhf/claimant/ClaimantLifecycleIntegrationTests.java

```diff
@@ -0,0 +1,230 @@
+        stubNotificationEmailResponse();
+
+        ClaimDTO claimDTO = aClaimDTOWithClaimant(aClaimantDTOWithExpectedDeliveryDate(expectedDeliveryDate));
+        restTemplate.exchange(buildClaimRequestEntity(claimDTO), ClaimResultDTO.class);
+        invokeAllSchedulers();
+
+        ClaimantDTO claimant = claimDTO.getClaimant();
+        Claim claim = repositoryMediator.getClaimForNino(claimant.getNino());
+        return claim.getId();
+    }
+
+    private List<LocalDate> fastForwardOneCycle(List<LocalDate> initialDates) throws JsonProcessingException {
+        repositoryMediator.fastForwardDatabaseEntities(CYCLE_DURATION);
+        List<LocalDate> newDates = initialDates.stream().map(localDate -> localDate.minusDays(CYCLE_DURATION)).collect(Collectors.toList());
+        wiremockManager.stubSuccessfulEligibilityResponse(newDates);
+        invokeAllSchedulers();
+        return newDates;
+    }
+
+    private void assertFirstCyclePaidCorrectly(UUID claimId) throws NotificationClientException {
+        Claim claim = repositoryMediator.loadClaim(claimId);
+        PaymentCycle paymentCycle = repositoryMediator.getCurrentPaymentCycleForClaim(claim);
+        assertThat(claim.getClaimStatus()).isEqualTo(ClaimStatus.ACTIVE);
+        assertPaymentHasCorrectAmount(claim, paymentCycle, NO_CHILDREN);
+        assertThatNewCardEmailSentCorrectly(claim, paymentCycle);
+        verifyNoMoreInteractions(notificationClient);
+    }
+
+    private PaymentCycle assertRegularPaymentCyclePaidCorrectly(UUID claimId, List<LocalDate> childrenDob) throws NotificationClientException {
+        Claim claim = repositoryMediator.loadClaim(claimId);
+        assertThat(claim.getClaimStatus()).isEqualTo(ClaimStatus.ACTIVE);
+        PaymentCycle paymentCycle = getAndAssertPaymentCycle(claim);
+        assertPaymentHasCorrectAmount(claim, paymentCycle, childrenDob);
+        assertThatPaymentEmailWasSent(paymentCycle);
+        return paymentCycle;
+    }
+
+    private void assertPaymentCycleWithBackdatedVouchersForNewChild(UUID claimId, List<LocalDate> childrenDob) throws NotificationClientException {
+        Claim claim = repositoryMediator.loadClaim(claimId);
+        assertThat(claim.getClaimStatus()).isEqualTo(ClaimStatus.ACTIVE);
+        PaymentCycle paymentCycle = getAndAssertPaymentCycle(claim);
+        PaymentCycleVoucherEntitlement expectedEntitlement =
+                aPaymentCycleVoucherEntitlementWithBackdatedVouchersForYoungestChild(paymentCycle.getCycleStartDate(), childrenDob);
+        assertThat(paymentCycle.getVoucherEntitlement()).isEqualTo(expectedEntitlement);
+        assertThat(paymentCycle.getPayments()).isNotEmpty();
+        Payment payment = paymentCycle.getPayments().iterator().next();
+        assertThat(payment.getPaymentAmountInPence()).isEqualTo(expectedEntitlement.getTotalVoucherValueInPence());
+        assertThatNewChildEmailWasSent(paymentCycle);
+    }
+
+    private PaymentCycle getAndAssertPaymentCycle(Claim claim) {
```

Comment:
Do you want to assert the payment cycle has the correct status here too? Seems prudent if you're using this to assert the payment cycle has been paid correctly.